vllm/v1/core/kv_cache_manager.py

```diff
@@ -110,39 +111,72 @@ def get_computed_blocks(
         num_computed_tokens = len(computed_blocks) * self.block_size
         return computed_blocks, num_computed_tokens
 
-    def append_slots(
-        self,
-        request: Request,
-        num_tokens: int,
+    def allocate_slots(
+        self, request: Request, num_tokens: int,
+        new_computed_blocks: List[KVCacheBlock]
```

Comment:
Use None as the default and have `computed_blocks = new_computed_blocks or []` in the beginning of the function, so that you don't need to pass `[]` everywhere

```suggestion
        new_computed_blocks: Optional[List[KVCacheBlock]] = None
```